azurerm/validators.go

```diff
@@ -74,3 +75,114 @@ func validateIso8601Duration() schema.SchemaValidateFunc {
 		return
 	}
 }
+
+func validateAzureTimeZone() schema.SchemaValidateFunc {
+	return validation.StringInSlice([]string{
```

Comment:
Could we please get a comment here linking to where this list is from? Are you sure it is accurate?